src/RuleSet.php

```diff
@@ -177,7 +177,16 @@ final class RuleSet implements RuleSetInterface
             'error_suppression' => true,
             'fopen_flag_order' => true,
             'fopen_flags' => ['b_mode' => false],
-            'function_to_constant' => true,
+            'function_to_constant' => [
+                'functions' => [
```

Comment:
Why updating the config in this ruleset? Was it asked by the Symfony community?

Reply:
Indeed; https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3900#issuecomment-523433454

Reply:
Ah thank you, I didn't read the whole discussion sorry :)

Reply:
all good, no worries :)